.github/workflows/cifuzz.yml

```
@@ -0,0 +1,26 @@
+name: CIFuzz
```

nit:
```suggestion
name: CI Fuzz
```
for consistency with other names in our CI